File: abc114/src/bin/b.rs
use proconio::{fastout, input};

#[fastout]
fn main() {
    input! {
        s: String,
    }
    let answer = (0..s.len() - 2)
        .map(|i| (s[i..i + 3].parse::<i32>().unwrap() - 753).abs())
        .min()
        .unwrap();
    println!("{}", answer);
}

File: abc114/b.ml
open Base;;

let s = Stdlib.read_line () in
let answer =
  List.range 0 (String.length s - 2)
  |> List.map ~f:(fun pos ->
         String.sub s ~pos ~len:3 |> Int.of_string |> ( - ) 753 |> abs)
  |> List.fold ~init:753 ~f:min
in
answer |> Int.to_string |> Stdlib.print_endline
